pkg/minishift/provisioner/utils.go

```diff
@@ -27,19 +27,8 @@ import (
 )
 
 var (
-	engineConfigTemplateRHEL = `[Unit]
-Description=Docker Application Container Engine
-Documentation=http://docs.docker.com
-After=network.target rc-local.service
-Requires=rhel-push-plugin.socket
-Requires=docker-cleanup.timer
-
-[Service]
-Type=notify
-NotifyAccess=all
-Environment=GOTRACEBACK=crash
-Environment=DOCKER_HTTP_HOST_COMPAT=1
-Environment=PATH=/usr/libexec/docker:/usr/bin:/usr/sbin
```

Comment:
> basically adding systemd files, but that's not required anymore and hence can be removed? Correct?

we still add some part of systemd files that why we didn't remove it completely but now they are consider as systemd drop-in units instead complete systemd file.